test/tools/router_check/json/tool_config_schemas.cc

```diff
@@ -0,0 +1,73 @@
+#include "test/tools/router_check/json/tool_config_schemas.h"
+
+const std::string& Json::ToolSchema::routerCheckSchema() {
+  static const std::string* router_check_schema = new std::string(R"EOF(
+  {
+    "$schema": "http://json-schema.org/schema#",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "properties": {
+          "test_name": {"type": "string"},
+          "input": {
+            "type": "object",
+            "properties": {
+              ":authority": {"type": "string", "required": true},
+              ":path": {"type": "string"},
+              ":method": {"type": "string", "enum": ["GET", "PUT", "POST"]},
+              "random_value": {"type": "integer"},
+              "ssl": {"type": "boolean"},
+              "internal": {"type": "boolean"},
+              "additional_headers": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "field": {"type": "string"},
+                    "value": {"type": "string"}
+                  },
+                  "additionalProperties": false,
+                  "required": ["field", "value"],
+                  "maxProperties": 2
```

Comment:
I put this here because the json schema allows properties to be repeated. For example: 

``{
  "field": ":authority",
  "value": "http://google.com",
  "field": ":path"
}``

is acceptable. By saying both fields are required and that only two properties are allowed, I am trying to deter json input files with erroneous schemas.